src/compiler/crystal/semantic/exception.cr

```diff
@@ -72,6 +72,17 @@ module Crystal
       end
     end
 
+    def inspect_with_backtrace(io : IO) : Nil
```

Comment:
I'm confused - this method is defined in `::Exception` and it doesn't seem to be overridden.

Reply:
What do you mean? It's overwritten here.

Reply:
I think RX14 wants to know you're overwritting it, doesn't the method in Exception already does that?

Reply:
The bodies are different: `::Exception` appends the `message`, but this one calls `to_s(io)`